furious/processors.py

```diff
@@ -67,7 +67,7 @@ def run_job():
     except AbortAndRestart as restart:
         logging.info('Async job was aborted and restarted: %r', restart)
         raise
-    except Exception as e:
+    except BaseException as e:
```

Comment:
The docs show that `google.appengine.runtime.DeadlineExceededError` inherits from BaseException:
https://code.google.com/p/googleappengine/source/browse/trunk/python/google/appengine/runtime/__init__.py

Also of note is the try/except block.